modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Dynamic Data Mapping API
 Bundle-SymbolicName: com.liferay.dynamic.data.mapping.api
-Bundle-Version: 6.4.1
+Bundle-Version: 6.5.0
```

Comment:
Why was it necessary, @marcosapmf? Probably, you needed to do that because your code wasn't synced with the lastest version of master, please check that the version 6.5.0 is already committed on master, as you can see [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/bnd.bnd). Can you please rebase this pull with the latest version of master?